packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -160,10 +169,22 @@ export default class GoogleCalendarService implements Calendar {
           dateTime: event.endTime,
           timeZone: event.organizer.timeZone,
         },
-        attendees: [{ ...event.organizer, organizer: true, responseStatus: "accepted" }, ...event.attendees],
+        attendees: [
+          {
+            ...event.organizer,
+            id: String(event.organizer.id),
+            organizer: true,
+            responseStatus: "accepted",
+          },
+          ...event.attendees.map(({ id, ...rest }) => ({
+            rest,
+            responseStatus: "accepted",
+          })),
```

Comment:
We don't need id here and not all attendees will have an id if they are new to the event

Reply:
Same here